sfbrigade/support-sfusd
# Integrate search functionality

Integrate search functionality into the map page by incorporating a search bar. This feature allows users to find specific schools by name.

As the user types, a dropdown menu will appear, providing potential matches. When the user selects a school, relevant information will be displayed in the adjacent SchoolCard on the map view.

In the list view, the selected school will be highlighted within the schools list.

After a school is selected, the search bar input will be automatically cleared to facilitate further searches.

![3](https://github.com/sfbrigade/support-sfusd/assets/88466266/171e5b33-c8a7-4edd-89c6-f11801ee6c44)
![5](https://github.com/sfbrigade/support-sfusd/assets/88466266/96b97872-3fd6-4ee9-afce-8830a4fc0496)

## Testing Steps 
git pull
git checkout feature/search-bar
npm install
npm run dev
Open http://localhost:3000/ with your browser to see the result
